molgenis-core/src/test/java/org/molgenis/util/MolgenisDateFormatTest.java

```diff
@@ -1,20 +1,63 @@
 package org.molgenis.util;
 
-import org.testng.Assert;
+import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
-import java.text.ParseException;
+import java.time.Instant;
 import java.time.LocalDate;
 import java.time.ZoneId;
-import java.util.Date;
+
+import static org.testng.Assert.assertEquals;
 
 public class MolgenisDateFormatTest
 {
 	@Test
-	public void getDateFormat() throws ParseException
+	public void testFormatLocalDate()
+	{
+		assertEquals(MolgenisDateFormat.getLocalDateFormatter().format(LocalDate.parse("2014-12-31")), "2014-12-31");
+	}
+
+	@Test
+	public void testFormatInstant()
+	{
+		assertEquals(MolgenisDateFormat.getDateTimeFormatter().format(Instant.parse("2000-12-31T12:34:56.789Z")),
+				"2000-12-31T12:34:56.789Z");
+	}
+
+	@Test
+	public void testGetDefaultZoneId()
+	{
+		assertEquals(MolgenisDateFormat.getDefaultZoneId(), ZoneId.of("Europe/Amsterdam"));
+	}
+
+	@DataProvider(name = "parseInstantDataProvider")
+	public Object[][] parseInstantDataProvider()
+	{
+		return new Object[][] { { "2000-12-31T12:34:56.789+0200", "2000-12-31T10:34:56.789Z" },
+				{ "2000-12-31T12:34:56.789+02:00", "2000-12-31T10:34:56.789Z" },
+				{ "2000-12-31T12:34:56.789", "2000-12-31T11:34:56.789Z" },
+				{ "2000-12-31T12:34", "2000-12-31T11:34:00Z" }, { "2000-12-31T12:34Z", "2000-12-31T12:34:00Z" },
+				{ "2000-12-31", "2000-12-30T23:00:00Z" }, };
```

Comment:
This last one's the most interesting one. We interpret incomplete date time strings as the start of date in the system's default time zone.